lms/djangoapps/instructor/views/api.py

```diff
@@ -2090,6 +2090,30 @@ def get_ora2_responses(request, course_id):
 @ensure_csrf_cookie
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 @require_level('staff')
+def get_course_forums_usage(request, course_id):
+    """
+    Pushes a Celery task which will aggregate course forum statistics into a .csv
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
+    try:
+        instructor_task.api.submit_course_forums_usage_task(request, course_key)
+        success_status = _("The course forum usage statistics is being generated.")
```

Comment:
it sounds more natural to say 'look at the forums' instead of 'look at the forum' . however, i don't have any strong preference and I'll follow up with course ops on this. thanks for pointing it out